Packages/com.unity.inputsystem/InputSystem/Editor/UITKAssetEditor/Views/ActionMapsView.cs

```diff
@@ -13,11 +13,9 @@ namespace UnityEngine.InputSystem.Editor
     internal class ActionMapsView : ViewBase<ActionMapsView.ViewState>
     {
         public ActionMapsView(VisualElement root, StateContainer stateContainer)
-            : base(stateContainer)
+            : base(root, stateContainer)
         {
-            m_Root = root;
-
-            m_ListView = m_Root?.Q<ListView>("action-maps-list-view");
+            m_ListView = rootElement?.Q<ListView>("action-maps-list-view");
```

Comment:
> This is a pattern in all the constructors. It probably doesn't matter at the moment but if accessing base within constructor (while constructing the object) can be avoided its usually good to avoid.

Interesting, is that a style thing? I assumed performance-wise it would be the same; my thinking was that it's better to have everything accessing rootElement since then it's consistent across the files (eg. it's more clear that the root in the constructor and in RedrawUI is the same). Happy to change, either way!

Reply:
Its probably just in the category nit-picking and could be ignored, but since there is already a local (potentially) in a register, why call into a getter that needs to fetch the reference from base?